src/components/proofs/ProofList.tsx

```diff
@@ -21,47 +21,50 @@ export default function () {
   const mainnetProofAddressesAvailableToCreate =
     useProofAddressesAvailableToCreate(Network.Mainnet)
 
+  const hasCompletedProofs = proofsCompleted.length > 0
+  const hasGoerliProofsToCreate =
+    goerliProofAddressesAvailableToCreate.length > 0
+  const hasMainnetProofsToCreate =
+    mainnetProofAddressesAvailableToCreate.length > 0
+
   const nothingToGenerate =
-    proofsCompleted.length === 0 &&
-    goerliProofAddressesAvailableToCreate.length === 0 &&
-    mainnetProofAddressesAvailableToCreate.length === 0
+    !hasCompletedProofs && !hasGoerliProofsToCreate && !hasMainnetProofsToCreate
 
   return (
     <>
       <Scrollbar>
         <div className={space('space-y-2')}>
-          <ProofSection title={<BodyText>Mainnet NFTs</BodyText>}>
+          <ProofSection
+            title={<BodyText>Mainnet NFTs</BodyText>}
+            hasContent={hasMainnetProofsToCreate}
+          >
             <ReadyERC721ProofsList network={Network.Mainnet} />
             {account && (
               <AvailableProofsList
                 proofs={mainnetProofAddressesAvailableToCreate}
                 network={Network.Mainnet}
               />
             )}
-            {nothingToGenerate && (
-              <HintCard small>
-                <HintText bold center>
-                  No NFTs to proof
-                </HintText>
-              </HintCard>
-            )}
           </ProofSection>
-          <ProofSection title={<BodyText>Goerli NFTs</BodyText>}>
+          <ProofSection
+            title={<BodyText>Goerli NFTs</BodyText>}
+            hasContent={hasGoerliProofsToCreate}
+          >
             <ReadyERC721ProofsList network={Network.Goerli} />
             {account && (
               <AvailableProofsList
                 proofs={goerliProofAddressesAvailableToCreate}
                 network={Network.Goerli}
               />
             )}
-            {nothingToGenerate && (
-              <HintCard small>
-                <HintText bold center>
-                  No NFTs to proof
-                </HintText>
-              </HintCard>
-            )}
           </ProofSection>
+          {nothingToGenerate && (
```

Comment:
It was repeatable code, so I extracted it, we can actually display `No mainnet NFTs to proof` and `No goerli NFTs to proof`, but I think we don't need this